wp-media/wp-rocket
# Closes #5126 - Fixes background image breaking for lazyload with native lazyload

## Description

This PR fixes the issue of adding rocket lazyload attributes on images that lazyloaded natively.

Fixes #5126

## Type of change


## Is the solution different from the one proposed during the grooming?

There is no grooming.

## How Has This Been Tested?
- Add add_filter( 'rocket_use_native_lazyload', '__return_true' ) to current theme functions.
- Add an element to a template with inline background-image style and check that image displays when lazyload is applied.

# Checklist:
